recipes/ligo.skymap/meta.yaml

```diff
@@ -0,0 +1,159 @@
+  build:
+    - {{ compiler('c') }}
+    - pkg-config
+  host:
+    - python
+    - setuptools
+    - pip
+    - numpy 1.14.*
+    - chealpix
+    - gsl
+    - openmp
+  run:
+    - chealpix
+    - gsl
+    - openmp
+    - python
+    - {{ pin_compatible('numpy') }}
+    - astroplan
+    - astropy-healpix >=0.3
+    - astropy >=3.1
+    - healpy
+    - h5py
+    - lalsuite >=6.49
+    - ligo-common
+    - ligo-gracedb >=2.0.1
+    - lscsoft-glue
+    - matplotlib >=2.1.0
+    - networkx
+    - pillow >=2.5.0
+    - ptemcee
+    - pytz
+    - reproject >=0.3.2
+    - scipy >=0.14
+    - seaborn >=0.8.0
+    - setuptools
+    - tqdm
+
+test:
+  imports:
+    - ligo.skymap
+    - ligo.skymap.bayestar
+    - ligo.skymap.bayestar.ez_emcee
+    - ligo.skymap.bayestar.filter
+    - ligo.skymap.bayestar.interpolation
+    - ligo.skymap.bayestar.ptemcee
+    - ligo.skymap.coordinates
+    - ligo.skymap.coordinates.detector
+    - ligo.skymap.coordinates.eigenframe
+    - ligo.skymap.distance
+    - ligo.skymap.extern
+    - ligo.skymap.extern.quantile
+    - ligo.skymap.healpix_tree
+    - ligo.skymap.io
+    - ligo.skymap.io.events
+    - ligo.skymap.io.events.base
+    - ligo.skymap.io.events.detector_disabled
+    - ligo.skymap.io.events.gracedb
+    - ligo.skymap.io.events.hdf
+    - ligo.skymap.io.events.ligolw
+    - ligo.skymap.io.events.magic
+    - ligo.skymap.io.events.sqlite
+    - ligo.skymap.io.fits
+    - ligo.skymap.io.hdf5
+    - ligo.skymap.kde
+    - ligo.skymap.moc
+    - ligo.skymap.plot
+    - ligo.skymap.plot.allsky
+    - ligo.skymap.plot.angle
+    - ligo.skymap.plot.backdrop
+    - ligo.skymap.plot.cmap
+    - ligo.skymap.plot.cylon
+    - ligo.skymap.plot.marker
+    - ligo.skymap.plot.poly
+    - ligo.skymap.plot.pp
+    - ligo.skymap.plot.setup_package
+    - ligo.skymap.plot.util
+    - ligo.skymap.postprocess
+    - ligo.skymap.postprocess.contour
+    - ligo.skymap.postprocess.ellipse
+    - ligo.skymap.postprocess.find_injection
+    - ligo.skymap.postprocess.util
+    - ligo.skymap.setup_package
+    - ligo.skymap.util
+    - ligo.skymap.util.file
+    - ligo.skymap.util.numpy
+    - ligo.skymap.util.sqlite
+  requires:
+    - pytest
+  commands:
+    - python -m pytest --pyargs "ligo.skymap"
+    - bayestar-localize-coincs --help
+    - bayestar-localize-lvalert --help
+    - bayestar-mcmc --help
+    - bayestar-realize-coincs --help
+    - bayestar-sample-model-psd --help
+    - ligo-skymap-combine --help
+    - ligo-skymap-contour --help
+    - ligo-skymap-from-samples --help
+    - ligo-skymap-plot --help
+    - ligo-skymap-plot-airmass --help
+    - ligo-skymap-plot-stats --help
+    - ligo-skymap-plot-volume --help
+    - ligo-skymap-stats --help
+
+about:
+  home: https://lscsoft.docs.ligo.org/ligo.skymap/
+  doc_url: https://lscsoft.docs.ligo.org/ligo.skymap/
+  dev_url: https://git.ligo.org/lscsoft/ligo.skymap
+  license: GPLv2
+  license_family: GPL
+  license_file: '{{ environ["RECIPE_DIR"] }}/LICENSE.rst'
```

Comment:
Def submit this upstream. I know some of these repos are not public.

Reply:
https://git.ligo.org/lscsoft/ligo.skymap/issues/10